File: apps/codemap-search/experiments/event-navigation-poc/language-corpus/fixtures/java/callbacks.java
class Router {
    Runnable primary, secondary;
    void setPrimary(Runnable cb) { this.primary = cb; } // store_primary
    void setSecondary(Runnable cb) { this.secondary = cb; } // store_secondary
    void firePrimary() { primary.run(); } // call_primary
    void fireSecondary() { secondary.run(); } // call_secondary
}
class Other {
    Runnable primary;
    void firePrimary() { primary.run(); } // call_other
}

File: apps/codemap-search/experiments/event-navigation-poc/language-corpus/fixtures/python/callbacks.py
class Router:
    def set_primary(self, cb):
        self.primary = cb  # store_primary
    def set_secondary(self, cb):
        self.secondary = cb  # store_secondary
    def fire_primary(self):
        self.primary()  # call_primary
    def fire_secondary(self):
        self.secondary()  # call_secondary
class Other:
    def fire_primary(self):
        self.primary()  # call_other
